requirements.txt

```diff
@@ -0,0 +1 @@
+pytest
```

Comment:
include `pandas` as it is used in tests

Reply:
Thanks; fixed in https://github.com/Electrostatics/propka-3.1/commit/6288711209a4f72c3dca2ec9f51b3bec5b6f8734